demos/4_Deep_Dives/03_htc_2022.ipynb

```diff
@@ -122,14 +124,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "\n",
     "# Load the full data:\n",
-    "full_data = load_htc2022data(\"./htc_code/data/htc2022_ta_full.mat\")\n",
+    "full_data = load_htc2022data(\"/mnt/share/materials/SIRF/Fully3D/CIL/HTC/htc2022_ta_full.mat\")\n",
```

Comment:
```suggestion
    "full_data = load_htc2022data(\"/mnt/materials/SIRF/Fully3D/CIL/HTC/htc2022_ta_full.mat\")\n",
```

Reply:
sorry think this was me adding the wrong path!

Reply:
Done